src/m365/commands/status.ts

```diff
@@ -23,36 +22,34 @@ class StatusCommand extends Command {
           await logger.logToStderr(err);
         }
 
-        auth.service.logout();
+        auth.service.deactivateIdentity();
         throw new CommandError(`Your login has expired. Sign in again to continue. ${err.message}`);
       }
 
+      const response = {
+        ...auth.getIdentityDetails(auth.service, this.debug),
+        connectedAs: auth.service.identityName // (Deprecated), added for backwards compatibility
+      };
+
       if (this.debug) {
-        await logger.logToStderr({
-          connectedAs: accessToken.getUserNameFromAccessToken(auth.service.accessTokens[auth.defaultResource].accessToken),
-          authType: AuthType[auth.service.authType],
-          appId: auth.service.appId,
-          appTenant: auth.service.tenant,
-          accessTokens: JSON.stringify(auth.service.accessTokens, null, 2),
-          cloudType: CloudType[auth.service.cloudType]
-        });
+        await logger.logToStderr(response);
       }
       else {
-        await logger.log({
-          connectedAs: accessToken.getUserNameFromAccessToken(auth.service.accessTokens[auth.defaultResource].accessToken),
-          authType: AuthType[auth.service.authType],
-          appId: auth.service.appId,
-          appTenant: auth.service.tenant,
-          cloudType: CloudType[auth.service.cloudType]
-        });
+        await logger.log(response);
```

Comment:
To stay consistent with all identity-related commands I changed the shape of login, status, identity set and list to be the same. Of course it is backwards compatible. Some props were added is all.

Reply:
I wonder if that's necessary and doesn't lead to outputting excessive information that's not needed in the context of each command. If you set an identity for example, do you expect the same input as when you call `status`?

Reply:
I would think that the output of `m365 identity set` should be the same as the output of `m365 status` and `m365 login`. There's a good reason to think a user might keep the output of one of those commands around to use it in scripts. It would be handy to be able to update your identity variable when running a set, without needing to run m365 status another time.

So I think that's quite optimal. 
The `m365 identity list` command could be trimmed down a bit of course. But I personally would think staying consistent is better here.

Reply:
What would be the reason for users to have to keep their identity in the script? I wonder if it's something that we should consider addressing differently rather than offloading everything to users. We've got for example the `@me` tokens that take a part of that responsibility away. Are we missing something?

Reply:
The reasons to do it could be many. For example: someone might be migrating stuff and logging lines with properties from the active identity in it. Who knows... It's just an example. 

I'd personally prefer the consistency. And we have many `set` commands that return an response, so I'm not sure why it would be a problem exactly.